src/js/wrap.h

```diff
@@ -243,6 +243,9 @@ namespace ccf::js
     std::chrono::milliseconds max_exec_time = default_max_execution_time;
 
   public:
+    bool log_exception_details = false;
+    bool return_exception_details = false;
```

Comment:
Does this have to be `public:`?

Reply:
It needs to be accessible by the frontend.